npm/qsharp/ux/circuit-vis/events.ts

```diff
@@ -420,7 +419,9 @@ class CircuitEvents {
     dropzoneElems.forEach((dropzoneElem) => {
       dropzoneElem.addEventListener("mouseup", async (ev: MouseEvent) => {
         const copying = ev.ctrlKey;
-        const originalGrid = cloneDeep(this.componentGrid);
+        const originalGrid = JSON.parse(
+          JSON.stringify(this.componentGrid),
+        ) as ComponentGrid;
```

Comment:
~What's the goal of stringifying then parsing?~   Ohh - is a deep clone? Neat trick but I've never seen it before - a quick comment would help confused people like me

Reply:
Oh this is actually used in a number of places in this code and from before we inherited it. I thought this was a standard, but still good to add a clarifying comment. Done!